src/vivarium/framework/randomness.py

```diff
@@ -649,11 +650,16 @@ def get_randomness_stream(self, decision_point: str, for_initialization: bool=Fa
             with the same identifier.
         """
         if decision_point in self._decision_points:
-            raise RandomnessError(f"Two separate places are attempting to create "
-                                  f"the same randomness stream for {decision_point}")
-        self._decision_points.add(decision_point)
-        return RandomnessStream(key=decision_point, clock=self._clock, seed=self._seed,
-                                index_map=self._key_mapping, manager=self, for_initialization=for_initialization)
+            if throw_error:
```

Comment:
Drop the throw error (revert to the old code) with just the change to a dict instead of a set.